### ISSUE TITLE
Add dir element in <p> tag automatically

### ISSUE THREAD

**Issue by rezaervani** [humhub/humhub#5678]
**How To Reproduce**

Based on article at : https://www.w3.org/International/questions/qa-html-dir

I did some experience to my humhub, add dir element in p tag on stream, it make every RTL text (arabic) automatically start from Right side, and keep LTR text start from left side

![Screenshot from 2022-05-04 07-02-54](https://user-images.githubusercontent.com/5996010/166605158-86dd0c6e-2d28-4765-8a8c-dc1bea13cb89.png)

With code is looked like this : 

![Screenshot from 2022-05-04 07-06-21](https://user-images.githubusercontent.com/5996010/166605281-4be0a3e4-27a9-4936-abc8-710445334865.png)

**Expected Result**

The Question is : 

How to make dir element automatically shown in stream (after type a status from markdown editor) ?

Thank you

**Comment by ArchBlood** [comment]
See #4875

**Comment by luke-** [comment]
Do I understand it correctly that with `<p dir="auto">` everything is displayed correctly? And it's just a matter of adding the `dir="auto"` attribute at the e.g. P?

**Comment by rezaervani** [comment]
> Do I understand it correctly that with `<p dir="auto">` everything is displayed correctly? 

Yes. All this time all stream text display from left, include RTL language like arabic, add dir=auto will correct it.

Based on the same article, the solution should not be done through CSS

**Comment by luke-** [comment]
Instead of adding an RTL attribute everywhere, could you not add here: 
https://github.com/humhub/humhub/pull/4875/files

**Comment by ArchBlood** [comment]
> Instead of adding an RTL attribute everywhere, could you not add here: 
> https://github.com/humhub/humhub/pull/4875/files
> 
> 
> 

Wouldn't work on some bootstrap elements

**Comment by luke-** [comment]
But on the Richtext?

**Comment by ArchBlood** [comment]
> But on the Richtext?

Adding auto would work only on text, menu icons and such will not be changed
![Screenshot_1](https://user-images.githubusercontent.com/35392110/167275307-0d3e3842-5766-477f-b26e-c3643284549d.png)

Richtext is however changed with or without auto
![Screenshot_2](https://user-images.githubusercontent.com/35392110/167275314-20c19b69-7099-4273-9aea-f23ad65dd404.png)

**Comment by ArchBlood** [comment]
It also happens with all RTL-supported languages of course.

**Comment by ArchBlood** [comment]
The only way I could think of fixing this without changing to a newer version of Bootstrap would be by making special containers for menus and dropdown menus where it would support the RTL functions.

**Comment by rezaervani** [comment]
> But on the Richtext?

I try this schema on prosemirror : 

<pre>
paragraph: {
    content: "inline*",
    group: "block",
    attrs: {
      dir: {default: "auto"},
    },
    parseDOM: [
      {
        tag: "p",
        getAttrs: (dom) => ({dir: dom.getAttribute("dir")}),
      },
    ],
    toDOM(node) {
      const {dir} = node.attrs;
      return ["p", {"dir": dir}, 0];
    },
  },
</pre>

it will add attribute dir=auto automatically in paragraph tag

So when I type arabic, it will start from right side.

I put rough demo here : https://rezaervani.com/koding/prose.html

![Screenshot from 2022-05-09 13-43-09](https://user-images.githubusercontent.com/5996010/167354625-f1262885-c819-457b-9edd-1d2f3069a293.png)
